src/polymer/polymer-core-feature.ts

```diff
@@ -12,9 +12,88 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {ScannedPolymerElement} from '../polymer/polymer-element';
+import {Class} from '../model/class';
+import {Document, Feature, Method, Property, Resolvable, ScannedFeature, ScannedMethod, ScannedProperty, Warning} from '../model/model';
 
 /**
- * The metadata for a Polymer core feature.
+ * A scanned Polymer 1.x core "feature".
  */
-export interface ScannedPolymerCoreFeature extends ScannedPolymerElement {}
+export class ScannedPolymerCoreFeature extends ScannedFeature implements
+    Resolvable {
+  warnings: Warning[] = [];
+  properties: ScannedProperty[] = [];
+  methods: ScannedMethod[] = [];
+
+  resolve(document: Document): Feature|undefined {
+    if (this.warnings.length > 0) {
+      return;
+    }
+
+    // Sniff for the root `_addFeatures` call by presence of this method. This
+    // will normally be in the `polymer.html` document.
+    const isRootAddFeatureCall =
+        this.methods.some((p) => p.name === '_finishRegisterFeatures');
```

Comment:
You confirmed that this is only present in the polymer core source in one place? Kevin wasn't sure that it was a good indicator